doc/wishlist.md

```diff
@@ -0,0 +1,41 @@
+| This is a living document. It describes the policy and priorities as they exist today, but can evolve over time. |
+|---|
+
+# Node.js Modules Group Wishlist
```

Comment:
Perhaps we can fold this into the roadmap document?

Reply:
The reason why I decided to separate them, other than my notion that a roadmap should be more narrowly scoped, is because the combination of them in one document appeared, from my perspective, to look like an enormous, incoherent wall of text and I found it unappealing. 

This might've been remedied with an improved list structure or similar content refactoring. 

Definitely open to better ideas or even if others have stronger preferences, I wouldn't mind changing it up.

Reply:
| This is a living document. It describes the policy and priorities as they exist today, but can evolve over time. |
|---|

# Node.js Modules Group Wishlist